schemas/queries/prometheus/query.cue

```diff
@@ -13,15 +13,16 @@
 
 package prometheus
 
-#datasource: {
-	kind: "PrometheusDatasource"
-}
-
-#query: {
-	kind: "PrometheusGraphQuery"
-	options: {
-		query:       string
-		min_step?:   =~"^(?:(\\d+)y)?(?:(\\d+)w)?(?:(\\d+)d)?(?:(\\d+)h)?(?:(\\d+)m)?(?:(\\d+)s)?(?:(\\d+)ms)?$"
-		resolution?: number
+spec: {
```

Comment:
why not having the same syntax from the Panel ? 

```cue
#spec: plugin {
   kind: "PrometheusTimeSeriesQuery"
}

Reply:
actually the panels should align on the queries when we'll move the always-common part to go structs as we discussed ;)

Reply:
aah ok ! Sorry I forgot things during time. I'm so old haha